### ISSUE TITLE
Sites randomly blocked even though timer has not finished.

### ISSUE THREAD

**Issue by wobbba** [proginosko/LeechBlockNG#229]
First off, I do not have the exact steps to reproduce. It appears random to me. 
I'm using Firefox on Windows. I have **a lot** of tabs opened (probably >1000) in a lot of windows. Also quite a few Extensions. 

I've only noticed it with YouTube but IDK if it is specific to the site. I have a block set that lets me watch YT (or prime/netflix, ...) for 50 min then it blocks it. But sometimes I looked at the timer 5 min ago and it said 25 min remaining when suddenly it blocks. I have a hunch that it may have something to do with multiple tabs of YT being opened and loaded (maybe in different windows or on different monitors) but I have not experimented thoroughly.

Very frustrating when I'm half-way through my lunch...

**Comment by proginosko** [comment]
Have you selected the option "Process only active tabs" (under General options)?

**Comment by wobbba** [comment]
yes. should I try unchecking it?

**Comment by proginosko** [comment]
No, you don't need to do that. With >1000 tabs open, your browser will probably grind to a halt. :smile: I was just curious. I would investigate whether its connected to having multiple YT tabs open.
